tasks/queue-granules/tests/index.js

```diff
@@ -102,10 +102,10 @@ test.serial('The correct output is returned when granules are queued without a P
   const { event } = t.context;
   event.input.granules = [
     {
-      dataType: dataType, version: version, granuleId: randomString(), files: []
+      dataType, version, granuleId: randomString(), files: []
```

Comment:
Not a question about this test but about the logic in queue-granules. So queue-granules does this

```
const collectionConfig = await collectionConfigStore.get(granule.dataType, granule.version);
```

Why is it pulling from granule.dataType and granule.version instead of getting those values from the configured collection name and version? I realize that right now queue-granules does not expect a collection as task config, but it could. It also feels odd to keep using `granule.dataType` when we mean `granule.collectionName`

Is the idea that queue-granules could handle queueing granules from multiple collections? Is that even possible though, given that discover-granules expects a single collection definition and triggers queue-granules?

Reply:
> Is the idea that queue-granules could handle queueing granules from multiple collections?

That's my understanding, yes.

Reply:
@markdboyd `queue-granules` is also used by the `ParsePdr` workflow, and a PDR _can_ contain multiple granules of differing collections.